stdio/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+extern crate futures;
+extern crate tokio;
+extern crate tokio_codec;
+extern crate tokio_io;
+extern crate tokio_stdin_stdout;
+
+pub extern crate jsonrpc_core;
+use jsonrpc_core::{IoHandler};
+
+use tokio::prelude::{Future, Stream};
+use tokio_codec::{FramedRead, FramedWrite, LinesCodec};
+
+fn process(io: &IoHandler, input: String) -> Box<Future<Item = String, Error = ()> + Send> {
+  Box::new(
+    io.handle_request(&input).map(move |result| match result {
+      Some(res) => res,
+      None => {
+        eprintln!("JSON RPC request produced no response: {:?}", input);
+        String::from("")
+      }
+    })
+  )
+}
+
+pub fn server(io: IoHandler) {
```

Comment:
Please add documentation to this function and `#![warn(missing_docs)]` to the entire crate.